src/Features/LanguageServer/ProtocolUnitTests/SemanticTokens/AbstractSemanticTokensTests.cs

```diff
@@ -104,5 +104,13 @@ protected static async Task VerifyNoMultiLineTokens(TestLspServer testLspServer,
                     $"The token ends at index {currentChar + tokenLength}, which exceeds the line length of {lineLength}.");
             }
         }
+
+        protected static LSP.SemanticTokensEdit GenerateEdit(int start, int deleteCount, int[] data)
```

Comment:
this method seems unnecessary.  seems equivalent to just `new SemanticTOkensEdit { ... }`